web/api/report_server.thrift

```diff
@@ -304,27 +304,28 @@ struct ReviewStatusRule {
 typedef list<ReviewStatusRule> ReviewStatusRules
 
 struct ReportData {
-  1: i64              runId,           // Unique id of the run.
-  2: string           checkerId,       // The qualified name of the checker that reported this.
-  3: string           bugHash,         // This is unique id of the concrete report.
-  4: string           checkedFile,     // This is a filepath, the original main file the analyzer was called with.
-  5: string           checkerMsg,      // Description of the bug report.
-  6: i64              reportId,        // id of the report in the current run in the db.
-  7: i64              fileId,          // Unique id of the file the report refers to.
-  8: i64              line,            // line number or the reports main section (not part of the path).
-  9: i64              column,          // column number of the report main section (not part of the path).
-  10: Severity        severity,        // Checker severity.
-  11: ReviewData      reviewData,      // Bug review status information.
+  1: i64              runId,              // Unique id of the run.
+  2: string           checkerId,          // The qualified name of the checker that reported this.
+  3: string           bugHash,            // This is unique id of the concrete report.
+  4: string           checkedFile,        // This is a filepath, the original main file the analyzer was called with.
+  5: string           checkerMsg,         // Description of the bug report.
+  6: i64              reportId,           // id of the report in the current run in the db.
+  7: i64              fileId,             // Unique id of the file the report refers to.
+  8: i64              line,               // line number or the reports main section (not part of the path).
+  9: i64              column,             // column number of the report main section (not part of the path).
+  10: Severity        severity,           // Checker severity.
+  11: ReviewData      reviewData,         // Bug review status information.
   12: DetectionStatus detectionStatus, // State of the bug (see the enum constant values).
   13: string          detectedAt,      // Detection date of the report.
   14: string          fixedAt          // Date when the report was fixed.
-  15: i64             bugPathLength,   // Length of the bug path.
-  16: optional ReportDetails details,  // Details of the report.
-  17: optional string analyzerName,    // Analyzer name.
+  15: i64             bugPathLength,  // Length of the bug path.
+  16: optional ReportDetails details, // Details of the report.
+  17: optional BlameInfo blameInfo,   // Blmae info.
```

Comment:
New parameters should be added to the end of the list without changing previous numbers. As far as I know, these numbers play a role in API backward compatibility.